napari/layers/_base_layer/model.py

```diff
@@ -6,10 +6,11 @@
 from skimage import img_as_ubyte
 
 from ...util.event import Event
+from ...util.keybindings import KeymapMixin
 from ._visual_wrapper import VisualWrapper
 
 
-class Layer(VisualWrapper, ABC):
+class Layer(VisualWrapper, KeymapMixin, ABC):
```

Comment:
Total side note: I'm also unclear from the name `VisualWrapper` what it is or what it's doing, so that might be another thing to rename in the near future.

Reply:
`VisualWrapper` relates to our handling of the `vispy` scene and will get pulled out when we separate our vispy code from our layer models